### ISSUE TITLE
Error in Julia v0.5

### ISSUE THREAD

**Issue by juliohm** [JuliaGPU/CLFFT.jl#12]
Can someone please fix this issue?

``` julia
using CLFFT
```

> WARNING: `@unix_only` is deprecated, use `@static if is_unix()` instead
>  in depwarn(::String, ::Symbol) at ./deprecated.jl:64
>  in @unix_only(::Any) at ./deprecated.jl:492
>  in include_from_node1(::String) at ./loading.jl:488 (repeats 2 times)
>  in eval(::Module, ::Any) at ./boot.jl:234
>  in require(::Symbol) at ./loading.jl:415
>  in eval(::Module, ::Any) at ./boot.jl:234
>  in eval_user_input(::Any, ::Base.REPL.REPLBackend) at ./REPL.jl:64
>  in macro expansion at ./REPL.jl:95 [inlined]
>  in (::Base.REPL.##3#4{Base.REPL.REPLBackend})() at ./event.jl:68
> while loading /home/juliohm/.julia/v0.5/CLFFT/src/api.jl, in expression starting on line 6
> ERROR: LoadError: LoadError: UndefVarError: CL_uint not defined
>  in include_from_node1(::String) at ./loading.jl:488 (repeats 2 times)
>  in eval(::Module, ::Any) at ./boot.jl:234
>  in require(::Symbol) at ./loading.jl:415
> while loading /home/juliohm/.julia/v0.5/CLFFT/src/api.jl, in expression starting on line 22
> while loading /home/juliohm/.julia/v0.5/CLFFT/src/CLFFT.jl, in expression starting on line 6

**Comment by dfdx** [comment]
Please, check out master, I think it should be already fixed there. If everything is fine, I'll tag a new version.

**Comment by juliohm** [comment]
Indeed, it is solved in master, could you please tag a new version?

**Comment by juliohm** [comment]
Before tagging a new version, please fix another warning:

> WARNING: deprecated syntax "d::Cint".
> Use "local d::Cint" instead.

Thank you.

**Comment by juliohm** [comment]
Another one that looks more serious:

> WARNING: call(f,args...) is deprecated, use f(args...) instead.
>  in depwarn(::String, ::Symbol) at ./deprecated.jl:64
>  in ##983#984(::Array{Any,1}, ::Function, ::OpenCL.cl.CmdQueue, ::OpenCL.cl.Kernel, ::Vararg{Any,N}) at ./deprecated.jl:271
>  in imfilter_gpu(::Array{Float64,3}, ::Array{Float64,3}, ::String) at /home/juliohm/.julia/v0.5/ImageQuilting/src/imfilter_gpu.jl:53
>  in #convdist#1(::BitArray{3}, ::Bool, ::Function, ::Array{Any,1}, ::Array{Any,1}) at /home/juliohm/.julia/v0.5/ImageQuilting/src/utils.jl:33
>  in (::ImageQuilting.#kw##convdist)(::Array{Any,1}, ::ImageQuilting.#convdist, ::Array{Any,1}, ::Array{Any,1}) at ./<missing>:0
>  in #iqsim#14(::Float64, ::Float64, ::Float64, ::Array{ImageQuilting.SoftData,1}, ::Dict{Tuple{Integer,Integer,Integer},Real}, ::Float64, ::Symbol, ::Symbol, ::Bool, ::Int64, ::Bool, ::Bool, ::ImageQuilting.#iqsim, ::Array{Float64,3}, ::Int64, ::Int64, ::Int64, ::Int64, ::Int64, ::Int64) at /home/juliohm/.julia/v0.5/ImageQuilting/src/iqsim.jl:299
>  in (::ImageQuilting.#kw##iqsim)(::Array{Any,1}, ::ImageQuilting.#iqsim, ::Array{Float64,3}, ::Int64, ::Int64, ::Int64, ::Int64, ::Int64, ::Int64) at ./<missing>:0
>  in include_string(::String, ::String) at ./loading.jl:441
>  in execute_request(::ZMQ.Socket, ::IJulia.Msg) at /home/juliohm/.julia/v0.5/IJulia/src/execute_request.jl:169
>  in eventloop(::ZMQ.Socket) at /home/juliohm/.julia/v0.5/IJulia/src/eventloop.jl:8
>  in (::IJulia.##9#15)() at ./task.jl:360
> while loading In[1], in expression starting on line 184

This is where I am calling it: https://github.com/juliohm/ImageQuilting.jl/blob/master/src/imfilter_gpu.jl#L53

**Comment by dfdx** [comment]
> WARNING: deprecated syntax "d::Cint".

Fixed in `other-deprecation-warnings` branch, I'll merge and tag when we don't have visible warnings anymore. 

> WARNING: call(f,args...) is deprecated, use f(args...) instead.

According to [OpenCL.jl new](https://github.com/JuliaGPU/OpenCL.jl/blob/55663bd9f0a2612b7a47968d1342b425abe7a0b6/NEWS.md), seems like this line should now look like: 

```
queue(mult_kernel, length(A), nothing, bufA, bufkrn, bufRES)
```

**Comment by juliohm** [comment]
Thank you, I fixed the problem with `cl.call`. Please close the issue when you merge the fixes for the warnings.

**Comment by dfdx** [comment]
Merged and tagged as 0.3.0.
